docs/variables/count.md

```diff
@@ -0,0 +1,97 @@
+---
+id: count
+---
+
+# $(count)
+
+Returns the number of times a command has been executed in a channel and optionally offers the ability to create custom counting variables.
+
+## $(count.get)
+
+Returns the current number that is stored in a custom counting variable.
+
+#### Parameters
+
+This variable takes **one** *required* parameter that is a custom counting variable name. Defaults to the behavior of `$(count)` if no parameter is provided, and returns as well as increments the command usage count by `1`.
+
+#### Example Output
+
+* `$(count.get my-cool-variable)`
+
+    ```
+    23
+    ```
+
+#### Error Output
+
+* In case an invalid custom counting variable name is provided, returns the following:
+
+    ```
+    [Error: Count name invalid. Must contain only alphanumeric, dashes, or underscores.]
+    ```
+
+## $(count.update)
```

Comment:
Even tho `$(count.increment)` behaves the same, I think we should list that too, as it would prob make more sense to most ppl

Reply:
I'll add a new list item for it.